src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -467,6 +684,12 @@ public SCMRevision getTrustedRevision(SCMRevision revision, TaskListener listene
         public static final String defaultExcludes = "";
         public static final String ANONYMOUS = "ANONYMOUS";
         public static final String SAME = "SAME";
+        public static final boolean defaultBuildOriginBranch = true;
+        public static final boolean defaultBuildOriginBranchWithPR = true;
+        public static final boolean defaultBuildOriginPRMerged = false;
+        public static final boolean defaultBuildOriginPRUnmerged = false;
+        public static final boolean defaultBuildForkPRMerged = true;
+        public static final boolean defaultBuildForkPRUnmerged = false;
```

Comment:
Matches current behavior: build fork PRs plus origin branches, and try to build a merge revision for PRs.